src/main/resources/sql/payment_terms/dev-pg-pt-ddl.sql

```diff
@@ -0,0 +1,4 @@
+create table payment_terms (days_until_due integer not null, creation_time timestamp(6) not null, id bigint generated by default as identity, modification_time timestamp(6), code varchar(255) not null, created_by varchar(255) not null, modified_by varchar(255), name varchar(255) not null, primary key (id), constraint uix_payment_terms_code unique (code));create table payment_terms (days_until_due integer not null, creation_time timestamp(6) not null, id bigint generated by default as identity, modification_time timestamp(6), code varchar(255) not null, created_by varchar(255) not null, modified_by varchar(255), name varchar(255) not null, primary key (id), constraint uix_payment_terms_code unique (code));
```

Comment:
do not commit this file as part of your final commit. The scripts should be moved to other project.